bundle/manifests/splunk-operator.clusterserviceversion.yaml

```diff
@@ -799,14 +802,14 @@ spec:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
                 - name: RELATED_IMAGE_SPLUNK_ENTERPRISE
-                  value: splunk/splunk:9.0.1-a1
+                  value: docker.io/splunk/splunk:edge
```

Comment:
Do we need this change?

Reply:
this is an autogenerated file. changes need to be set to 9.1 splunk release

Reply:
we need to run 
`
make bundle IMAGE_TAG_BASE=docker.io/splunk/splunk-operator VERSION=2.1.0 IMG=splunk/splunk-operator:2.1.0 SPLUNK_ENTERPRISE_IMAGE=splunk/splunk:9.1
`